packages/kbn-ecs/package.json

```diff
@@ -2,5 +2,6 @@
   "name": "@kbn/ecs",
   "version": "1.0.0",
   "private": true,
-  "license": "SSPL-1.0 OR Elastic License 2.0"
+  "license": "SSPL-1.0 OR Elastic License 2.0",
+  "sideEffects": false
```

Comment:
**note:** required to tree-shake the package and prevent importing +100kb of unused code in async chunks.